src/main/kotlin/io/github/inductiveautomation/kindling/internal/FileTransferHandler.kt

```diff
@@ -7,7 +7,7 @@ import java.io.IOException
 import javax.swing.TransferHandler
 
 class FileTransferHandler(
-    private val predicate: (File) -> Boolean,
+    private val predicate: (File) -> Boolean = { true },
```

Comment:
Can you explain why this change was necessary?

Reply:
Since there was no predicate before the tile change was made, this mimics the old behavior of FileTransferHandler before it had a predicate.

If preferred, I can remove the default value & pass it in manually.

Reply:
Ah, got it, now I'm following. No, this is fine.